build/target-repository/bootstrap.php

```diff
@@ -14,7 +14,7 @@
     }
 
     // load prefixed or native class, e.g. for running tests
-    if (str_starts_with($class, 'ECSPrefix') || str_starts_with($class, 'Symplify\\')) {
+    if (strpos($class, 'ECSPrefix') === 0 || strpos($class, 'Symplify\\') === 0) {
```

Comment:
This line can be replaced with:

```suggestion
    if (strncmp($class, 'ECSPrefix', strlen('ECSPrefix')) === 0 || strncmp($class, 'Symplify\\', strlen('Symplify\\')) === 0) {
```

`strlen('Symplify\\')` is replaced by constant `9` by the PHP compiler, so performance isn't too bad. https://3v4l.org/G0mkd/vld
However, the class name is not a very long string, so it looks like an over-optimization.